samples/petstore/microservices/pet/src/main/java/org/wso2/msf4j/examples/petstore/pet/Runner.java

```diff
@@ -24,11 +24,17 @@
  * Microservice runner for the pet microservices.
  */
 public class Runner {
+
+    private Runner() {
+    }
+
     public static void main(String[] args) {
+        JWTSecurityInterceptor jwtSecurityInterceptor = new JWTSecurityInterceptor();
+        HTTPMonitoringInterceptor httpMonitoringInterceptor = new HTTPMonitoringInterceptor();
+        MetricsInterceptor metricsInterceptor = new MetricsInterceptor();
         new MicroservicesRunner()
-                .addInterceptor(new JWTSecurityInterceptor())
-                .addInterceptor(new HTTPMonitoringInterceptor())
-                .addInterceptor(new MetricsInterceptor())
+                .addGlobalRequestInterceptor(jwtSecurityInterceptor, httpMonitoringInterceptor, metricsInterceptor)
```

Comment:
addInterceptor() method is deprecated. Samples are moved to use the new interceptor (excluding analytics samples)